src/System.Runtime.InteropServices/tests/Configurations.props

```diff
@@ -2,8 +2,11 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
-      netcoreapp;
+      netcoreapp-Windows_NT;
+      netcoreapp-Unix;
```

Comment:
>So I think this is right, but is there any value in the OS suffixes for the uap/uapaot ones?

OS suffixes should be avoided unless there is a reason that we need to condition code for a given OS. From a quick look at the code in this PR I don't think we need OS suffixes for any of the configurations. I don't see any #ifdefs or conditioned source files based on Windows or Unix for netcoreapp so they shouldn't be needed.